addons/website_sale/models/product.py

```diff
@@ -441,12 +441,7 @@ def _compute_product_website_url(self):
     @api.multi
     def website_publish_button(self):
         self.ensure_one()
-        res = self.product_tmpl_id.website_publish_button()
-
-        # res can be the result of write() if not website_publisher
-        if type(res) == dict and res.get('type') == 'ir.actions.act_url':
-            res['url'] = self.website_url
-        return res
+        return self.product_tmpl_id.website_publish_button()
```

Comment:
Is the right website_url still used for product variants ?

Reply:
since `website_publish_button` does not use website_url anymore it is good like this but `open_website_url` is the one that will need to use the new url. I'll add it